tests/test_training.py

```diff
@@ -44,6 +44,7 @@ def test_training(datastore_name):
         # because it expects to aggregate over multiple devices
         devices=2,
         log_every_n_steps=1,
+        detect_anomaly=True,
```

Comment:
Maybe we could add a comment to explain what this is for?

```suggestion
        # use `detect_anomaly` to ensure that we don't have NaNs popping up during training
        detect_anomaly=True,
```

Reply:
Done. Should I just go ahead and merge?